_docs/login-functionality.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Login Functionality
+permalink: /docs/publisher-tutorials/login-functionality/
+---
+
+# Login Functionality
+
+Currently there are two third party application to login with: 
+- GitHub 
+- Google
+
+If you have no Dockstore accounts, click either of the "Login with" buttons to create a new Dockstore account.  The username will be auto-generated.
+
+At this point, you will be able to link as many tokens as you want (Quay.io, Bitbucket, GitLab, Google/GitHub).  If the first Dockstore account was created with GitHub and then a Google token was linked (i.e. you have a Google and GitHub token linked to the same account), you are then able to login to that same account with either "Login with Google" or "Login with GitHub".  
```

Comment:
1. There is a slash separating Google/GitHub because they're currently our only two ways of creating a Dockstore account.  If you follow the steps previously.  One of those two tokens (accounts) would've automatically be linked.
2. See below